e2e/scripts/st_plotly_chart.py

```diff
@@ -42,7 +42,9 @@
 
 # Bubble Chart
 # Tests Discrete coloring with streamlit theme
-st.plotly_chart(fig_bubble, theme="streamlit")
+# check that use_container_width sets width even when height and width are specified
```

Comment:
"should use container width when flag is set, even when width and height are specified"